qdrant-landing/content/documentation/cloud/authentication.md

```diff
@@ -5,31 +5,46 @@ weight: 30
 
 # Database Authentication in Qdrant Managed Cloud
 
-This page shows you how to use the Qdrant Cloud Console to create a Database API key for a cluster. You will learn how to connect to your cluster using the new API key.
+This page describes what Database API keys are and shows you how to use the Qdrant Cloud Console to create a Database API key for a cluster. You will learn how to connect to your cluster using the new API key.
+
+Database API keys can be configured with granular access control. Please refer to the [Table of access](/documentation/guides/security/#table-of-access) to understand what permissions you can configure.
 
 ## Create Database API keys
 
-The API key is only shown once after creation. If you lose it, you will need to create a new one. 
-However, we recommend rotating the keys from time to time. To create additional API keys do the following.
+<aside role="alert">
+    The API key is only shown once after creation. If you lose it, you will need to create a new one. 
+</aside>
 
 1. Go to the [Cloud Dashboard](https://qdrant.to/cloud).
 2. Go to the **API Keys** section of the Cluster detail page.
-3. Click **Create**. If you have granular access control activated, you can now configure the permissions for the new key.
-4. Click **OK** and retrieve your API key. 
+3. Click **Create**.
+4. Choose a name and an optional expiration (in days, the default is 90 days) for your API key. An empty expiration will result in no expiration.
+5. The default permissions will result in cluster-wide manage/write permissions. Disabling manage/write permissions will result in read-only permissions.<br/><br/>If you want to configure access to only specific collections, you can select Collection. The collections available in your cluster will be shown to select and configure permissions.
+6. Click **Create** and retrieve your API key.
+
+We recommend configuring an expiration and rotating your API keys regularly as a security best practice.
+
+## Admin Database API keys
+
+The previous iteration of Database API keys, called Admin Database API keys, does not have granular access control. Clusters created before $GO_LIVE_DATE will still see the option to create Admin Database API keys. Admin Database API keys will remain working and will not be deleted but we do recommend to enable and use Database API keys with granular access control to take advantage of better security controls.
```

Comment:
'did' for past tense or 'do' for present tense plural  instead of 'does'

I prefer did instead of do but given they are still available do might work better.

Reply:
I think 'do' is better